commit e336b716cce83b3af414a1dcc4c2ef22c591d73f
Author: Christine Caulfield <ccaulfie@redhat.com>
Date:   Fri Feb 24 16:36:35 2017 +0000

    loop: Also set signals changed in qb_loop_signal_mod() back to SIG_DFL
    
    Signed-off-by: Christine Caulfield <ccaulfie@redhat.com>

diff --git a/lib/loop_poll.c b/lib/loop_poll.c
index 386cae6..cd2474e 100644
--- a/lib/loop_poll.c
+++ b/lib/loop_poll.c
@@ -723,6 +723,7 @@ qb_loop_signal_mod(qb_loop_t * lp,
 	sig->p = p;
 
 	if (sig->signal != the_sig) {
+		signal(sig->signal, SIG_DFL);
 		sig->signal = the_sig;
 		_adjust_sigactions_(s);
 	}
